alert-policies/node-js/LowApdexScore.yml

```diff
@@ -1,7 +1,7 @@
 name: Apdex Score
 
 description: |+
-  This alert is triggered when the Apdex score is below 0.5 for 5 minutes
+  This alert is triggered when the Apdex score is down for at least 5 minutes.
 type: STATIC
 nrql:
   query: "SELECT apdex(duration, t: 0.5) FROM Transaction WHERE appName like '%'"
```

Comment:
should this be `WHERE appName` or `FACET appName` ?

Reply:
WHERE appName